index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Apple</title>
+  <link href="styles/style.css" rel="stylesheet" type="text/css">
+  <link rel="icon" href="img/apple-icon.ico" />
+  <meta charset="utf-8"/>
+</head>
+
+<body>
+  <div class="top-content">
+    <div class="flex-spacer-1"></div>
+    <nav class="flex-spacer-1">
+      <ul>
+        <li>
+          <div><button class="round-first-button"><img class="button-apple-logo" src="img/globalnav_text.svg" alt="apple-logo"></button>
+          </div>
+        </li>
+        <li>
+          <div><button><img class="button-store" src="img/globalnav_text.svg" alt="store"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-mac" src="img/globalnav_text.svg" alt="mac"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-ipod" src="img/globalnav_text.svg" alt="ipod"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-iphone" src="img/globalnav_text.svg" alt="iphone"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-ipad" src="img/globalnav_text.svg" alt="ipad"></button></div>
+        </li>
```

Comment:
- [ ] Do not use screenshots to bypass requirements. Ensure to write the text on the background image only. You can use developer tools to grab the image/font-family from the original website and use normal `text` and `CSS` for the rest.

Reply:
I'm not using screenshots. If you check the archive website, this text is actually an image. There's no way in the world other than checking font families one by one to know which font apple used for this titles, so I just downloaded the .svg files and used them myself.

If the requirements of the project said somewhere "use text for the titles of the page", I would have done so, but apple was clearly using .svg files to do this, so I just copied what they did.